codegen/smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/core/KotlinSymbolProvider.kt

```diff
@@ -75,31 +78,29 @@ class KotlinSymbolProvider(private val model: Model, private val settings: Kotli
 
     override fun longShape(shape: LongShape): Symbol = numberShape(shape, "Long", "0L")
 
-    override fun floatShape(shape: FloatShape): Symbol = numberShape(shape, "Float", "0.0f")
+    override fun floatShape(shape: FloatShape): Symbol = numberShape(shape, "Float", "0f")
 
     override fun doubleShape(shape: DoubleShape): Symbol = numberShape(shape, "Double", "0.0")
 
     private fun numberShape(shape: Shape, typeName: String, defaultValue: String = "0"): Symbol =
-        createSymbolBuilder(shape, typeName, namespace = "kotlin")
-            .defaultValue(defaultValue)
-            .build()
+        createSymbolBuilder(shape, typeName, namespace = "kotlin").defaultValue(defaultValue).build()
 
     override fun bigIntegerShape(shape: BigIntegerShape?): Symbol = createBigSymbol(shape, "BigInteger")
 
     override fun bigDecimalShape(shape: BigDecimalShape?): Symbol = createBigSymbol(shape, "BigDecimal")
 
     private fun createBigSymbol(shape: Shape?, symbolName: String): Symbol =
-        createSymbolBuilder(shape, symbolName, namespace = "java.math", boxed = true).build()
+        createSymbolBuilder(shape, symbolName, namespace = "java.math", nullable = true).build()
 
     override fun stringShape(shape: StringShape): Symbol = if (shape.isEnum) {
         createEnumSymbol(shape)
     } else {
-        createSymbolBuilder(shape, "String", boxed = true, namespace = "kotlin").build()
+        createSymbolBuilder(shape, "String", nullable = true, namespace = "kotlin").build()
```

Comment:
**nit**: It would be nice if the PR only contained changes related to the feature/refactor. This appears like the base branch isn't up to date with `main` but `HEAD` is?